cloudsync_onedrive.py

```diff
@@ -309,15 +193,14 @@ class OneDriveProvider(Provider):         # pylint: disable=too-many-public-meth
         scopes=['profile', 'openid', 'email', 'files.readwrite.all', 'sites.readwrite.all', 'offline_access'],
     )
 
-    _additional_invalid_characters = '#'
-
     def __init__(self, oauth_config: Optional[OAuthConfig] = None):
         super().__init__()
         self._creds: Optional[Dict[str, str]] = None
         self.__cursor: Optional[str] = None
-        self.__client: onedrivesdk.OneDriveClient = None
+        self.__client: object = None
```

Comment:
is it helpful to typehint this as `object`?

Reply:
changed to a bool flag